### PR TITLE
Ingo horizontal calendar

### PR DESCRIPTION
**Pull request by varunon9**

### CODE REVIEW

File: src/calendar/index.js [hunk 1]
```diff
@@ -357,6 +363,65 @@ class Calendar extends Component {
     return CalendarHeader;
   }
 
+  showLoader() {
+    if (this.props.LoaderComponent) {
+      return <LoaderComponent />;
```
Comment (Abhijeetjaiswalit) [inline src/calendar/index.js:368]: this way, `LoaderComponent` will be undefined.
It should be `return this.props.LoaderComponent()`

thoughts?

Comment (varunon9) [reply src/calendar/index.js:368]: yeah, seems like i made a mistake. will have to check

Comment (Abhijeetjaiswalit) [reply src/calendar/index.js:368]: will you plz correct it whenever you get time? plz...

Comment (varunon9) [reply src/calendar/index.js:368]: Please create a PR. I'll merge it

**Comment by Abhijeetjaiswalit** [comment]
while pushing the changes, its giving the error =>

remote: Permission to varunon9/react-native-toggle-calendar.git denied to
Abhijeetjaiswalit.

File: src/calendar/index.js (hunk 1 above)
Comment (varunon9) [reply src/calendar/index.js:368]: Fork the repo, make changes to your forked repo and then you can create PR

**Pull request merged**